e2e/help/help.go

```diff
@@ -157,6 +175,31 @@ func (c *ctx) testSingularity(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+
+		printSuccessOrFailureFn := func(t *testing.T, r *e2e.SingularityCmdResult) {
+			if r.Stdout != nil {
+				t.Logf(string(r.Stdout) + "\n")
+			}
+			if r.Stderr != nil {
+				t.Logf(string(r.Stderr) + "\n")
+			}
+		}
+
+		e2e.RunSingularity(t, tt.name, e2e.WithArgs(tt.argv...),
+			e2e.PostRun(func(t *testing.T) {
```

Comment:
no need to call `PostRun` and checks, `ExpectExit` can take care of that, you just need to replace `shouldPass bool` at https://github.com/sylabs/singularity/blob/4e794e3b0c283d88558e160a7b790553d655d319/e2e/help/help.go#L169
 by `exit int` to provide the expected exit code